boards/wiznet/w5500_evb_pico2/board.yml

```diff
@@ -0,0 +1,6 @@
+board:
+  name: w5500_evb_pico2
+  full_name: W5500 Evaluation Pico2
```

Comment:
W5500-EVB-Pico2

Same mistake was made for the Pico-based board -- this should be the commercial name